src/MercadoPago/Client/Payment/PaymentPointOfInteractionRequest.cs

```diff
@@ -11,5 +11,15 @@ public class PaymentPointOfInteractionRequest
         /// Linked To.
         /// </summary>
         public string LinkedTo { get; set; }
+
+        /// <summary>
+        /// Type.
+        /// </summary>
+        public string Type { get; set; }
```

Comment:
esta propriedade pode ser nula
```suggestion
        public string? Type { get; set; }
```

Reply:
Conversei com o Danilo, e realmente era o que a IDE estava retornando de erro mesmo. Antes da versão 8.0 do C#, só pode ser usado o nullable em tipos primitivos. Logo, a string pode ser nula, mas não temos como setar isso na declaração da propriedade.